op-acceptance-tests/acceptance-tests.yaml

```diff
@@ -7,14 +7,26 @@
 
 
 gates:
-  - id: conductor
-    description: "Sanity/smoke acceptance tests for networks with conductors."
+  - id: isthmus
+    inherits:
+      - base
+    description: "Isthmus network tests."
     tests:
-      - package: github.com/ethereum-optimism/optimism/op-acceptance-tests/tests/base/conductor
+      - package: github.com/ethereum-optimism/optimism/op-acceptance-tests/tests/isthmus
+        timeout: 6h
+      - package: github.com/ethereum-optimism/optimism/op-acceptance-tests/tests/isthmus/operator_fee
+        timeout: 6h
+      - package: github.com/ethereum-optimism/optimism/op-acceptance-tests/tests/isthmus/withdrawal_root
+        timeout: 20m
+      - package: github.com/ethereum-optimism/optimism/op-acceptance-tests/tests/isthmus/erc20_bridge
+        timeout: 10m
+      - package: github.com/ethereum-optimism/optimism/op-acceptance-tests/tests/isthmus/pectra
         timeout: 10m
```

Comment:
Should we also include `preinterop` here? It's a directory within the 'isthmus' one.
@ajsutton , @Inphi

Reply:
Thank you! resolving to merge